lib/client/profile.go

```diff
@@ -449,6 +449,36 @@ func (p *ProfileStatus) DatabaseCertPathForCluster(clusterName string, databaseN
 	return keypaths.DatabaseCertPath(p.Dir, p.Name, p.Username, clusterName, databaseName)
 }
 
+// OracleWalletDir returns path to the specified database access
+// certificate for this profile, for the specified cluster.
+//
+// It's kept in <profile-dir>/keys/<proxy>/<user>-db/<cluster>/dbname-wallet/
+//
+// If the input cluster name is an empty string, the selected cluster in the
+// profile will be used.
+func (p *ProfileStatus) OracleWalletDir(clusterName string, databaseName string) string {
+	if clusterName == "" {
+		clusterName = p.Cluster
+	}
+
+	if path, ok := p.virtualPathFromEnv(VirtualPathDatabase, VirtualPathDatabaseParams(databaseName)); ok {
+		return path
+	}
```

Comment:
how does the virtual path work? would this conflict with `p.DatabaseCertPathForCluster`?

Reply:
The virtual path allows to spepcify the cert directory by env variables. Thus Oracle wallet uses the diferrent extensions  it should not conflict with `p.DatabaseCertPathForCluster`